.github/workflows/stage-build.yml

```diff
@@ -73,9 +73,6 @@ jobs:
     environment: stage
     runs-on: ubuntu-latest
 
-    # We only ever want to deploy the `next` branch to stage.
-    if: ${{ github.repository == 'mdn/yari' && github.ref_name == 'next' }}
-
```

Comment:
Nit: Should probably revert, this was to enforce using the `next` branch, to avoid accidentally overwriting each other's changes.